wolfcrypt/src/asm.c

```diff
@@ -445,7 +445,7 @@ __asm__(                           \
    "paddq %%mm5,%%mm3    \n\t" \
    "movd %%mm3,28(%0)    \n\t" \
    "psrlq $32, %%mm3     \n\t" \
-:"=r"(_c) : "0"(_c), "r"(tmpm) );
```

Comment:
Why was this reverted back to "g"? There is a comment below of why it had been changed to "r".

Reply:
I will revert this.